### PR TITLE
Fix #870: Lowfi Landscape HomeActivity

### LINKED ISSUE #870: Lowfi Landscape HomeActivity
Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0dbdd000-6738-4f6b-9026-0d16737b7baa/HP-Home-Page-

Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/ce2a96de-2c98-41f9-91e6-6ca0d8135fb2/HP-Home-Page-Card-3-

Add new test cases for landscape mode.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes #870 

Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0dbdd000-6738-4f6b-9026-0d16737b7baa/HP-Home-Page-

Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/ce2a96de-2c98-41f9-91e6-6ca0d8135fb2/HP-Home-Page-Card-3-

To test this you can inject `StoryProgressTestHelper` class in `HomeFragmentPresenter` and use following functions one by one. Make sure to uninstall your app if you are changing functions from `StoryProgressTestHelper`.
First Run - Without using `StoryProgressTestHelper` - you will see two `PromotedStory`.
Second Run - use markRecentlyPlayedForFractionsStory0Exploration0 from `StoryProgressTestHelper` - you will see one `PromotedStory`
Third Run - use markRecentlyPlayedForFirstExplorationInAllStoriesInFractionsAndRatios from `StoryProgressTestHelper` - you will see three `PromotedStory`.

Reference code: https://gist.github.com/rt4914/5258375c8d2cca81b6a5b62ccab8e143#file-homefragmentpresenter-kt-L92 (Line 92-93)

Also, #872 issues has been filed to make sure that in future we follow some standard format for naming ids too the views. For this PR it has not been fixed as it does not harm the code in any way at the same time if we made that change it would introduce changes in more number of files.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout-land/topic_summary_view.xml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:clickable="true"
+    android:elevation="4dp"
+    android:focusable="true"
+    android:onClick="@{(v) -> viewModel.clickOnSummaryTile()}"
+    app:roundedRectDrawableWithColor="@{viewModel.backgroundColor}">
+
+    <ImageView
+      android:id="@+id/imageView"
```
Comment (veena14cs) [inline app/src/main/res/layout-land/topic_summary_view.xml:23]: Change this id.

**Review by veena14cs — CHANGES_REQUESTED** [review]
One of the test case is failing testHomeActivity_recyclerViewIndex1_promotedCard_storyNameIsCorrect() please check.

**Comment by rt4914** [comment]
> testHomeActivity_recyclerViewIndex1_promotedCard_storyNameIsCorrect

@veena14cs all the test cases are passing. Can you please rerun them and if they fail can you post the error.

**Comment by veena14cs** [comment]
> > testHomeActivity_recyclerViewIndex1_promotedCard_storyNameIsCorrect
> 
> @veena14cs all the test cases are passing. Can you please rerun them and if they fail can you post the error.

It is passing now. May be I had some build issue.

**Review by veena14cs — APPROVED** [review]
LGTM.

**Pull request merged**